web/client/plugins/MapCatalog.jsx

```diff
@@ -67,7 +81,7 @@ const MapCatalogComponent = ({
 
 export default createPlugin('MapCatalog', {
     component: connect(createStructuredSelector({
-        active: state => state.controls?.mapCatalog?.enabled,
+        active: state => state.controls && state.controls.mapCatalog && state.controls.mapCatalog.enabled,
```

Comment:
This is ok, but why did you removed `?.` notation ? Did you find some problems? We updated babel to translate it.
I noticed it becomes a little unreadable in debug, maybe for this reason?

Reply:
After running npm run doctest I got jsdoc parsing error "Unexpected token"